### PR TITLE
New package: KPN.HIPINv4 version 4.3.2.6316

### PR DESCRIPTION
**Pull request by xhoy**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/148634)

### CODE REVIEW

**Comment by xhoy** [comment]
@microsoft-github-policy-service agree

**Comment by xhoy** [comment]
So, tried a local install and that worked great. No issues. Then i did an uninstall and i get an error

````
PS C:\temp\test\manifests> winget install --manifest .\k\KPN\HIPINv4\4.3.2.6316\
Found HIPIN v4 [KPN.HIPINv4] Version 4.3.2.6316
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.voipit.nl/HIPIN/HIPIN.exe
  ██████████████████████████████   167 MB /  167 MB
Successfully verified installer hash
Starting package install...
Successfully installed
PS C:\temp\test\manifests> get-package -name *HIPIN*

Name                           Version          Source                           ProviderName
----                           -------          ------                           ------------
HIPIN v4                       4.3.2.6316                                        Programs


PS C:\temp\test\manifests> winget uninstall --manifest .\k\KPN\HIPINv4\4.3.2.6316\
Found HIPIN v4 [KPN.HIPINv4]
No installed package found matching input criteria.
PS C:\temp\test\manifests> get-package -name *HIPIN*

Name                           Version          Source                           ProviderName
----                           -------          ------                           ------------
HIPIN v4                       4.3.2.6316                                        Programs
`````

After starting the application once i can do an uninstall via 
````
get-package -name *HIPIN* |uninstall-package 
````
So something is wrong with the uninstaller? I cannot find ANY documentation on how the manifest works. 

Other question, how do i create a dependency for the Microsoft.VCRedist.2015+.x64 & Microsoft.VCRedist.2015+.x86 package?

File: manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+PrivacyUrl: https://mondago.com/privacy-policy/
+Author: Mondago
+PackageName: HIPIN v4
+PackageUrl: https://www.mondago.com
+License: Copyright
+LicenseUrl: https://mondago.com/terms-and-conditions/
+Copyright: Copyright (c) KPN. All rights reserved.
+ShortDescription: HIPIN v4
+Description: HIPIN Voip software voor KPN/Routit voip telephone systems
+Moniker: HIPIN
```
Comment (Trenly) [inline manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.locale.en-US.yaml:18]:
```suggestion
Moniker: hipin
```

File: manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: KPN.HIPINv4
```
Comment (Trenly) [inline manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.installer.yaml:4]:
```suggestion
PackageIdentifier: KPN.HIPIN
```
Versions generally do not need to be included in the package identifier

File: manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: KPN.HIPINv4
```
Comment (Trenly) [inline manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: KPN.HIPIN
```

File: manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: KPN.HIPINv4
```
Comment (Trenly) [inline manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.yaml:4]:
```suggestion
PackageIdentifier: KPN.HIPIN
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: KPN.HIPIN
```
Comment (stephengillie) [inline manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.installer.yaml:4]: Please update file name and path to match this change.

File: manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: KPN.HIPIN
```
Comment (stephengillie) [inline manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.locale.en-US.yaml:4]: Please update file name and path to match this change.

File: manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: KPN.HIPIN
```
Comment (stephengillie) [inline manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.yaml:4]: Please update file name and path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @xhoy,

Automatic Validation ended with:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
'manifests\k\KPN\HIPINv4\4.3.2.6316', expected 
'manifests\k\KPN\HIPIN\4.3.2.6316' 

Currently, the file names and paths have an extra `v4`. They have to match the latter part of the `PackageIdentifier`.

**Comment by xhoy** [comment]
> Hi @xhoy,
> 
> Automatic Validation ended with:
> 
> > Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual
> > 'manifests\k\KPN\HIPINv4\4.3.2.6316', expected
> > 'manifests\k\KPN\HIPIN\4.3.2.6316'
> 
> Currently, the file names and paths have an extra `v4`. They have to match the latter part of the `PackageIdentifier`.

Fixed!

**Comment by stephengillie** [comment]
Verify: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**